packages/docker-proxy/docker-proxy.spec

```diff
@@ -8,7 +8,7 @@
 %global _dwz_low_mem_die_limit 0
 
 Name: %{_cross_os}docker-proxy
-Version: 19.03.13
+Version: 19.03.14
```

Comment:
It won't let me comment on an unchanged line above, but I think we have to update the libnetwork version as well, any time we update docker.  https://github.com/moby/moby/blob/v19.03.14/vendor.conf shows commit 55e924b8a84231a065879156c0de95aefc5f5435, I believe, and above we're on an older 026aabaa.

Reply:
Good catch